client/src/components/Comparison/ProductList/ProductList.jsx

```diff
@@ -86,14 +87,14 @@ class ProductList extends React.Component {
   render() {
     const { leftOffset, isRightScrollerShown } = this.state;
     const {
-      productsIdToDisplay, updateProductId, listType, productId,
+      productsIdToDisplay, updateProductId, listType, productId, isAdding
     } = this.props;
     const relatedProductExist = productsIdToDisplay.length !== 0;
     return (
       <div className={productList.products}>
-        <div className={productList.title} data-testid="title">
+        <h2 className={productList.title} data-testid="title">
           {listType}
```

Comment:
I'd recommend putting the text here so that there's spacing in the text. 

Also, "Your Products" should actually say "Your Outfit", right?

The text should also be uppercase to conform to the mock-up.